File: URI/1036.cpp
#include<iostream>
#include<cmath>
using namespace std;

int main(){
	double A, B, C, delta, R1, R2;

	cin >> A >> B >> C;

	delta = (B * B) - 4 * A * C;

	if(delta < 0.0 || A == 0.0){
		cout << "Impossivel calcular" << endl;
	}else{
		R1 = (-B + sqrt(delta))/ (2 * A);
		R2 = (-B - sqrt(delta))/ (2 * A);

		cout.precision(5);
		cout.setf(ios::fixed);
		cout << "R1 = " << R1 << endl;
		cout << "R2 = " << R2 << endl;
	}

	return 0;
}

File: URI/Python/1036.py
import math
lista = input().split()
a = float(lista[0])
b = float(lista[1])
c = float(lista[2])

delta = (b**2) - 4*a*c 

if a == 0.0 or delta < 0.0:
	print("Impossivel calcular")
else:
	r1 = (-b + math.sqrt(delta))/(2*a)
	r2 = (-b - math.sqrt(delta))/(2*a)
	print("R1 = {:.5f}".format(r1))
	print("R2 = {:.5f}".format(r2))

